packages/test-cli/sample-tests/module.test.mjs

```diff
@@ -0,0 +1,13 @@
+import assert from "node:assert";
+
+console.log("hewwo?");
```

Comment:
Should this still be here?

Reply:
yeah; that's testing that `console.log`s during test loading get printed to the screen

Reply:
(notably, that verbose mode indicates which file is running prior to the `console.log`s from that file appearing)